### PR TITLE
Dot descriptions complete

### LINKED ISSUE #138: Update dot descriptions on Explorer page
- Remove the dots from the right side of the filters panel, and instead show a dot by the label of a selected filter above its search box
- Change the dot to a more intuitive icon (question mark, or i )
- Update descriptions to be clearer

### PR DESCRIPTION
**Pull request by ryanctan**
Fixes #138.

1. Description icons changed to question marks
2. Descriptions moved from sidebar to at top of active filter
3. Description text added

Also, label corrections that apparently were not pushed were made (explore.js)

### CODE REVIEW

File: public/explore/explore.pug [hunk 1]
```diff
@@ -54,9 +45,18 @@
...
-                    date-search(data-title="{{dimension.label}}", data-field="{{dimension.field}}")
+                    date-search(data-title=" {{dimension.label}} ", data-field=" {{dimension.field}} ")
+
+                div(ng-mouseover="tooltip = dimension.field", style="float: right; position: relative; bottom: 324px; right: 7px; cursor: pointer;")
+                    span(ng-mouseover="tooltip = dimension.field", style="color: #227AB9; font-weight: bold") ?
+                    div.mentioned-names-popup(
+                          ng-mouseleave="(tooltip == dimension.field) && (tooltip = null)"
+                          ng-style="{'margin-left': '0px', 'margin-top': '-20px', display: 'block', opacity: tooltip == dimension.field ? '1': '0', 'z-index': tooltip == dimension.field ? 99: -1}"
+                    )
+                        //i(class="fa fa-times",style="top:10px;right:10px;position:absolute",ng-click="(tooltip == dimension.field) && (tooltip = null)")
```
Comment (epicfaace) [inline public/explore/explore.pug:57]: Can you remove this line?

**Review by epicfaace — CHANGES_REQUESTED** [review]
Looks good. Can you change it so that the tooltip comes to the _right_ of the question mark, rather than covering up the question mark? That way, it's a smoother experience for the user.

image

**Comment by ryanctan** [comment]
Changes made!

**Comment by ryanctan** [comment]
I'm gonna make a new branch, I accidentally committed non-relevant files.

**Pull request closed without merging**